service/worker/diagnostics/workflow.go

```diff
@@ -125,9 +124,8 @@ func (w *dw) DiagnosticsWorkflow(ctx workflow.Context, params DiagnosticsWorkflo
 		StartToCloseTimeout:    time.Second * 5,
 	}
 	activityCtx := workflow.WithActivityOptions(ctx, activityOptions)
-	timeoutsResult.Runbooks = []string{linkToTimeoutsRunbook}
 
-	err := workflow.ExecuteActivity(activityCtx, w.identifyIssues, identifyIssuesParams{
+	err := workflow.ExecuteActivity(activityCtx, identifyIssuesActivity, identifyIssuesParams{
```

Comment:
nit: This is equivalent since if the activity is registered by name, it will be called by name even if you use coderef.